packages/ERTP/src/issuer.js

```diff
@@ -22,11 +22,12 @@ import './types';
 function makeIssuerKit(
   allegedName,
   amountMathKind = MathKind.NAT,
-  displayInfo = undefined,
```

Comment:
Please don't change the name of this parameter. For one thing, it's in the documentation as the old way, so we would need to change quite a bit, and second, @erights prefers setting the same variable to a new value after coercion rather than having two names. It's something we've discussed for quite a bit on the ERTP side.

Reply:
Okay, will do.  In that case, I will need to add an additional `assert(displayInfo);` after the coercion or else Typescript doesn't narrow the type to say it cannot be `undefined`.